Engine/ScriptAnalyzerEngine.csproj

```diff
@@ -44,6 +48,9 @@
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\..\..\..\..\..\fbl_srv2_ci_mgmt.binaries.amd64chk\monad\System.Management.Automation.dll</HintPath>
```

Comment:
Yes. this came from my original checkin. I took a dependency on sma.dll from sourcedepot.